### DISCUSSION TITLE
Use cases for food hierarchy?

### DISCUSSION THREAD

**Discussion by beedaddy** [TandoorRecipes/recipes#1359] [Q&A] [answered]
When I'm on /list/food/ and click on »Move« I can select a parent food. I'm not sure what the idea behind that feature is. Perhaps someone can give me one or two use cases for this?

**Comment by smilerz** [comment, marked as answer]
Further enhancements are planned, but for now:

- foods can be set to inherit Shopping Category and Ignore Shopping
- in search when you select a keyword/food the returned recipes are those contain the food/keyword along with the food/keyword children

**Comment by tomtjes** [comment]
- Previously, I tagged recipes with seasons, depending on the ingredients: spring, summer,.... I'm considering to add seasons to the ingredients hierarchy instead, in order to find recipes with produce that's currently in season. (Caveat: many fruits and veg are in season for multiple seasons...)

- I created `dairy` and `meat` branches in my ingredient hierarchy, so that in the future, I might be able to find dairy-free or meat-free recipes. (I don't think it's currently possible to exclude ingredients in search)

- I want to cook something with cheese, but I don't care if the recipe requires Gouda, Emmentaler or Gruyere: A search for "cheese" gets me there if that's the parent node of the different varieties.

**Reply by beedaddy** [reply]
Very interesting. Thank you both.

**Comment by smilerz** [comment]
A couple future enhancements that hierarchies will help enable: 

- substitution (children/siblings are assumed substitutes)
- diet inheritance (food [meat] is restricted from diet vegetarian)
